shell/platform/android/io/flutter/view/AccessibilityBridge.java

```diff
@@ -539,6 +546,26 @@ private boolean shouldSetCollectionInfo(final SemanticsNode semanticsNode) {
                 accessibilityFocusedSemanticsNode, o -> o.hasFlag(Flag.HAS_IMPLICIT_SCROLLING)));
   }
 
+  @TargetApi(31)
+  @RequiresApi(31)
+  private void setBoldTextFlag() {
+    if (rootAccessibilityView == null || rootAccessibilityView.getResources() == null) {
+      return;
+    }
+    int fontWeightAdjustment =
+        rootAccessibilityView.getResources().getConfiguration().fontWeightAdjustment;
+    boolean shouldBold =
+        fontWeightAdjustment != Configuration.FONT_WEIGHT_ADJUSTMENT_UNDEFINED
+            && fontWeightAdjustment >= 300;
```

Comment:
Why 300? is there documentation around this value? This should be a constant for readability.

Reply:
Added a comment
300 is bold - normal = w700 - w400

Reply:
Can you also add a constant for the value 300?
```
static final int kBoldTextAdjustment = 300
```